<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="videoWidget" native="true">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionPlay"/>
   <addaction name="actionVideoOptions"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>169</width>
     <height>368</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="windowTitle">
    <string>Video Options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Subtitle</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QComboBox" name="comboBoxSub">
          <item>
           <property name="text">
            <string>No Subtitle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Load Subtitle</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButtonFont">
             <property name="text">
              <string>Font</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonColor">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSliderSubSize">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Delay</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Playback</string>
       </property>
       <layout class="QFormLayout" name="formLayout_5">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Speed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxSpeed">
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="keyboardTracking">
              <bool>true</bool>
             </property>
             <property name="suffix">
              <string> %</string>
             </property>
             <property name="minimum">
              <number>40</number>
             </property>
             <property name="maximum">
              <number>800</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Aspect Ratio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxAspect">
             <item>
              <property name="text">
               <string>Auto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1:1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3:2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4:3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5:4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14:9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14:10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16:9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16:10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2.35:1</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>A/V Delay</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxAVDelay">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="specialValueText">
              <string/>
             </property>
             <property name="accelerated">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string> sec</string>
             </property>
             <property name="minimum">
              <number>-500</number>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Loop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxLoop">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Audio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_9" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>&lt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="text">
    <string>Mute</string>
   </property>
  </action>
  <action name="actionVideoOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>VideoOptions</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
